### ISSUE TITLE
Test Recent Changes Page Interactive Elements for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4916]
# Page to Test:

Recent Changes - https://openlibrary.org/recentchanges

# Steps

Steps for this task are outlined in #4897
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Sabreen-Parveen** [comment]
@bpmcneilly I would like on this page.

**Comment by shaiksahilrizwan** [comment]
Hi, @bpmcneilly  @RayBB  I’d like to work on this issue. Could you please assign it to me?

I have manually tested the **"Recent Changes"** page using keyboard navigation (Tab / Tab + Shift) as outlined in #4897. Everything works great—the focus order is logical and elements are accessible.

I also ran pages through WAVE Tool (Scored 9.1/10). I noticed a couple of things that might be out of scope for this issue, but I wanted to check how you’d like me to handle them:

1. **Contrast:** WAVE flagged 2 contrast errors.  
2. **Semantic HTML:** The "Search By" dropdown beside search bar has an orphaned `<label>` (WAVE flagged this) and `<select>` inside it is missing `id/name` (checked through inspecting). Screen reader will correctly read this out to user because it has `aria-label`.

Should I open separate bugs for these, or can we consider this page’s testing completed?

**Comment by RayBB** [comment]
@shaiksahilrizwan thanks for testing this.

The search by dropdown is not part of this particular issue and I'll mention the contrast issue to the team but it's generally known and to be worked on. So for now we can close it. Thanks very much. I hope you can take on another issue from the epic as well! Would you like to? Perhaps https://github.com/internetarchive/openlibrary/issues/4913

**Comment by shaiksahilrizwan** [comment]
> [@shaiksahilrizwan](https://github.com/shaiksahilrizwan) thanks for testing this.
> 
> The search by dropdown is not part of this particular issue and I'll mention the contrast issue to the team but it's generally known and to be worked on. So for now we can close it. Thanks very much. I hope you can take on another issue from the epic as well! Would you like to? Perhaps [#4913](https://github.com/internetarchive/openlibrary/issues/4913)

Sounds good, thanks for the explanation! and I would love to take on #4913. Heading over there now.
